dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -99,6 +97,13 @@ def __init__(self,
         self._data_batches = [[None] for i in range(self._num_gpus)]
         self._counter = 0
         self._current_data_batch = 0
+        self._output_names_map = [x[0] for x in output_map]
+        self._output_categories_map = [x[1] for x in output_map]
```

Comment:
You can do this here:
```
self._output_names_map, self._output_categories_map = zip(*output_map)
```

Reply:
I never liked this zip(*something) tricks, as they can be harder to comprehend. I'll lave it as it is now.